spec/delete_host_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'rails_helper' # spec_helper?
+require 'sidekiq/testing'
+
+describe DeleteHost do
+  let(:user) { FactoryBot.create(:user) }
+  let(:host) { FactoryBot.create(:host, org_id: user.account.org_id) }
+  let(:profile) { FactoryBot.create(:profile, :with_rules, account: user.account) }
+  let(:message) { { 'id' => host.id, 'type' => 'delete' } }
+  let!(:test_result) { FactoryBot.create(:test_result, profile: profile, host: host) }
+  let!(:rule_result) do
+    FactoryBot.create(:rule_result, rule: profile.rules.sample, test_result: test_result, host: host)
+  end
+
+  before do
+    logger = double(Rails.logger)
+    allow(logger).to receive(:info)
+    allow(logger).to receive(:debug)
+    allow(Sidekiq).to receive(:logger).and_return(logger)
+
+    DeleteHost.clear
+  end
+
+  it 'performs and logs deletion' do
+    expect(profile.hosts.count).not_to eql(0)
+    expect(profile.score).not_to eql(0.0)
+    DeleteHost.perform_async(message)
+
+    expect(profile.test_results.count).to eql(1)
+    expect(profile.rule_results.count).to eql(1)
+    expect(Rails.logger).to receive(:audit_success).with("Deleted related records for host #{host.id}")
+    DeleteHost.drain
```

Comment:
Put a single empty line around these two, so they are logically separated from the expectations.